modules/reference/partials/platform-nav.adoc

```diff
@@ -0,0 +1,8 @@
+ 
```

Comment:
For concision's sake, I think what we'd expect to see in the TOC is

* Platforms 
** xref:platform/MicroProfile.adoc[MicroProfile]
**xref:platform/JavaEE.adoc[Java EE]
** xref:platform/JakartaEE.adoc[Jakarta EE]

@cbridgha @hlhoots - what do you think? I could make that edit manually for this release